zend_operator.go

```diff
@@ -32,7 +28,7 @@ type Floats interface {
 // is recommended for higher accuracy.
 // Reference :
 //   - https://github.com/php/php-src/blob/php-5.6.40/Zend/zend_operators.c#L627-L633
-func floatToString[T Floats](value T) string {
+func floatToString[T float32 | float64](value T) string {
 	f64 := float64(value)
```

Comment:
Why don't we just change the type parameter into float64 (and let caller do the float32 -> float64 casting)?

```suggestion
func floatToString(f64 float64) string {
```

Reply:
Refactored to accept only float64 and Updated the `ConvertToString`function to convert float32 variable to
float64 before calling the `floatToString` function.